docs/Components/Aerodynamic/RectangularHorizontalStabilizer.md

```diff
@@ -0,0 +1,19 @@
+## RectangularHorizontalStabilizer
+This class creates a rectangular horizontal stabilizer
+```python
+from vec import Vector2
+import math
+import pytest
+from adr.Components.Aerodynamic import RectangularHorizontalStabilizer
+
+rectangularHorizontalStabilizer = RectangularHorizontalStabilizer(
```

Comment:
In python, [we use "snake case" to name variables](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names).

rectangularHorizontalStabilizer would become rectangular_horizontal_stabilizer

CamelCase is just used for class names.